src/DataFactory/DataFactoryV2/help/Set-AzDataFactoryV2.md

```diff
@@ -232,6 +250,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -UserAssignedIdentitiesDefinition
```

Comment:
Please add description about `UserAssignedIdentity` and `IdentityType` and remove `UserAssignedIdentitiesDefinition`.